code/game/objects/items/explosives/grenades/grenade.dm

```diff
@@ -92,3 +92,54 @@
 		return
 	walk(src, null, null)
 	return
+
+
+////RAD GRENADE - TOTALLY RAD MAN
+
+/obj/item/explosive/grenade/rad
+	name = "\improper V-40 rad grenade"
+	desc = "Rad grenades release an extremely potent but short lived burst of radiation, debilitating organic life and frying electronics in a moderate radius. After the initial detonation, the radioactive effects linger for a time. Handle with extreme care."
+	icon_state = "grenade_rad" //placeholder
+	item_state = "grenade_rad" //placeholder
+	icon_state_mini = "grenade_red" //placeholder
+	det_time =  40 //default
+	arm_sound = 'sound/weapons/armbomb.ogg' //placeholder
+	hud_state = "grenade_he" //placeholder
+	///The range for the grenade's full effect
+	var/inner_range = 4
+	///The range range for the grenade's weak effect
+	var/outer_range = 7
+	///The potency of the grenade
+	var/rad_strength = 20
+	///geiger counter sound loop
+	var/datum/looping_sound/geiger/geiger_counter
```

Comment:
I dont see the sound getting nulled or deleted

Reply:
Moved the var